<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>820</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>toDev</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="actionsGroup">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>350</width>
      <height>510</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="createBtn">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>150</y>
       <width>90</width>
       <height>90</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="openBtn">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>270</y>
       <width>90</width>
       <height>90</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="createLabel">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>240</y>
       <width>90</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Создать</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="openLabel">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>360</y>
       <width>90</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Открыть</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="projectLabel">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>351</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Проект...&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="parameterGroup">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>40</y>
      <width>350</width>
      <height>510</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="settingsBtn">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>200</y>
       <width>90</width>
       <height>90</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="createLabel_2">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>290</y>
       <width>90</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Настройки</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
